### PR TITLE
Improve documentation writing

### PR DESCRIPTION
**Pull request by LouisLeNezet**
During the review of the phaseimpute pipeline a few comments was done on portions of the documentation originating from the template.
Here is the improved documentation.

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: nf_core/pipeline-template/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# {{ name }}: Contributing Guidelines
+# `{{ name }}`: Contributing Guidelines
```
Comment (LouisLeNezet) [inline nf_core/pipeline-template/.github/CONTRIBUTING.md:1]: I'm not sure if by adding the '`' the `{{name}}` will still be changed.

Comment (ewels) [reply nf_core/pipeline-template/.github/CONTRIBUTING.md:1]: That should be fine, it won't cause any problems.

File: nf_core/pipeline-template/.github/CONTRIBUTING.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ These tests are run both with the latest available version of `Nextflow` and als
 
 - On your own fork, make a new branch `patch` based on `upstream/master`.
 - Fix the bug, and bump version (X.Y.Z+1).
-- A PR should be made on `master` from patch to directly this particular bug.
+- A PR should be made on `master` from patch to directly adress this particular bug.
```
Comment (ewels) [inline nf_core/pipeline-template/.github/CONTRIBUTING.md:69]:
```suggestion
- A PR should be made on `master` from patch to directly address this particular bug.
```

Comment (ewels) [reply nf_core/pipeline-template/.github/CONTRIBUTING.md:69]: Actually, maybe better:
```suggestion
- Open a pull-request from `patch` to `master` with the changes.
```

File: nf_core/pipeline-template/docs/usage.md [hunk 3]
```diff
@@ -110,37 +109,34 @@ nextflow pull {{ name }}
...
-:::info
-We highly recommend the use of Docker or Singularity containers for full pipeline reproducibility, however when this is not possible, Conda is also supported.
-:::
+> [!IMPORTANT]
+> We highly recommend the use of Docker or Singularity containers for full pipeline reproducibility, however when this is not possible, Conda is also supported.
 
 {%- if nf_core_configs %}
 
-The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
+The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to check if your system is suported, please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
```
Comment (mashehu) [inline nf_core/pipeline-template/docs/usage.md:139]:
```suggestion
The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to check if your system is supported, please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
```

**Review by mashehu — APPROVED** [review]

**Pull request merged**